common/culture/traditions/00_combat_traditions.txt

```diff
@@ -1187,8 +1197,10 @@ tradition_malleable_invaders = {
 	
 	is_shown = {
 		has_dlc_feature = hybridize_culture
+		
+		# Warcraft
 		OR = {
-			has_cultural_tradition = tradition_seafaring
+			heritage_gives_sea_raiding_trigger = yes
 			draenor_culture_trigger = yes
 		}
```

Comment:
Don't get why generic tradition is locked behind our conditions.
I wouldn't do it.

Reply:
if `tradition_seafaring` is the condition, then it means cultures with `heritage_gives_sea_raiding_trigger = yes` will not be able to pick `tradition_malleable_invaders`, which makes no sense. This is because `tradition_seafaring` is only shown to `heritage_gives_sea_raiding_trigger  = no` due to the heritage raiding effects.